backend/api/main.py

```diff
@@ -530,10 +551,16 @@ async def get_match_status(
         raise HTTPException(status_code=404, detail="Datapoint not found!")
 
     async with aiohttp.ClientSession() as session:
-        response = await session.get(
-            f"{ORION_URL}/v2/entities/{row['entity_id']}/attrs/{row['attribute_name']}/?type={row['entity_type']}"
-        )
-        return response.status == 200
+        entity_id = row['entity_id']
```

Comment:
Where is now the checking function? I though it will be either in API or in Gateway, but I could not find it

Reply:
fixed, I have created a separate function in main.py which check it now